### PR TITLE
DO-384-Loader Message(23 June)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,11 +10,12 @@
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
   <meta name="apple-mobile-web-app-title" content="listanyone">
   <link rel="apple-touch-icon" href="https://directory-online-qa.s3.ap-south-1.amazonaws.com/default/images/listanyone-lcon.png">
+  <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@400;600;700&display=swap" rel="stylesheet">
```
Comment (mohitsaladiya) [inline src/index.html:13]: Remove this. We can't import a font just for displaying text.

File: src/index.html [hunk 2]
```diff
@@ -10,11 +10,12 @@
...
   <style>
     body{
       margin: 0;
       padding: 0;
-      font-family: Nunito Sans, sans-serif;
+      font-family: 'Nunito Sans', sans-serif!important;
```
Comment (mohitsaladiya) [inline src/index.html:18]: Remove this line from body. As we should not apply font-family on body with an !important tag.

Add this font-family in  .loading-text-content class.

 .loading-text-content {
  font-family: sans-serif !important;
}

Comment (Palakagrawal28) [reply src/index.html:18]: Done

File: src/index.html (hunk 1 above)
Comment (Palakagrawal28) [reply src/index.html]: Done

**Pull request merged**